src/pages/Results.pug

```diff
@@ -109,9 +109,50 @@ block article
         This concludes, that our fusion protein was not present in the samples and the conjugation did not work well enough to be able to perform a cyclic synthesis with the immobilized primers. 
         We decided to switch to the next immobilization method. 
         
-        ### Immobilization with Biotinylated Primer
+        ### Immobilization with Biotin-labelled Primer
+        
+        Our third immobilization method was a combination of streptavidin-tagged magnet beads and biotin-labelled primers.
```

Comment:
Diesen Einleitungsabschnitt kann man evtl auch weglassen, weil das auch in Experiments steht. Schadet aber auch nicht denke ich :)